spec/helpers/feature_flags_helper_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the FeatureFlagsHelper. For example:
+#
+# describe FeatureFlagsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe FeatureFlagsHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
why do we need this?

Reply:
File was created automatically, is there need to remove this file?

Reply:
If you don't use it, than remove